### DISCUSSION TITLE
Issue with address field, can it be fixed?

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5632] [Q&A] [answered]
We are occasionally getting a message in where the PID-11.2 is missing. 

`|220 W SOUTH B ST^GAS CITY^IN^46933-1719`

I would normally expect this:

`|220 W SOUTH B ST^^GAS CITY^IN^46933-1719`

Mirth interprets GAS CITY as the PID-11.2 in the first line. I'm not sure if there is a way to detect when this is happening and insert the missing caret.

**Comment by pacmano1** [comment, marked as answer]
Sure, you have to write code to detect and fix it by whatever algorithm makes sense for your data.    Or pester the sender to fix their side.
